src/pages/blog/video-banner/_DBanner.js

```diff
@@ -126,25 +144,19 @@ const Dbanner = ({ video_details, video_list }) => {
                                 color="grey-17"
                                 width="auto"
                             >
-                                {video_details[0].duration}
+                                {video_duration}
                             </Header>
                         </Flex>
                     </Flex>
                     <VideoCarousel carousel_items={video_list} />
                 </Container>
             </ParentWrapper>
-            {show && (
-                <VideoPlayer
-                    video_src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerEscapes.mp4"
-                    closeVideo={handleCloseVideo}
-                />
-            )}
+            {show && <VideoPlayer video_src={video_url} closeVideo={handleCloseVideo} />}
```

Comment:
```suggestion
            {show && <VideoPlayer video_src={`${video_url}.mp4`} closeVideo={handleCloseVideo} />}
```

Reply:
If not mistaken, you will need to add the extension based on the video type for it to work.

Reply:
Sure @sean-binary ~ I'll confirm this and do the appropriate changes